### ISSUE TITLE
overview of the development status of the PermaplanT project

### ISSUE THREAD

**Issue by HappyPony** [ElektraInitiative/PermaplanT#340]
it would be advantageous if a potential interested party or potential contributor, but also the already involved contributor could get an overview of the development status of the PermaplanT project in a time-saving way.

Is there an overview, where roughly ON A SHEET the contents of PermaplanT project are visualized, for example:
- which PermaplanT modules, subprojects does the PermaplanT project currently have?
- the development status of the currently developed modules?

**Comment by markus2330** [comment]
Yes, I agree we must lower entry barriers. Right now we are in a too early stage: maintaining such a list would be too much work currently, there are too many moving parts. Even the structure is not fixed yet (e.g. #326). What is "in progress" can be seen:

- in the **Status** field of the [usecases](https://github.com/ElektraInitiative/PermaplanT/tree/master/doc/usecases)
- in the [project](https://github.com/orgs/ElektraInitiative/projects/4)

At the moment it is best that potential contributors directly write us an [email](mailto:contact@permaplant.net), saying what their skills are and how they would be willing to help.
